wiki/Help_centre/Website/ro.md

```diff
@@ -0,0 +1,104 @@
+
+Dacă dorești să ignori mesaje evidențiate, mesaje private sau mesaje publice, poți adăuga pur și simplu `@h`, `@p`, `@c` la numele lor de utilizator. `The_Gatekeeper@c` va ignora mesajele utilizatorului în canalele publice. Poți să le combini: `The_Gatekeeper@ph` va ignora mesajele evidențiate și mesajele private, dar mesajele publice rămân vizibile.
+
+#### Blocarea tuturor străinilor de a trimite mesaje private în joc {id=ignore-strangers-in-game}
+
+Dacă dorești să împiedici persoanele care nu se află pe lista ta de prieteni să îți trimită mesaje private, urmează următorii pași. **Ține cont: moderatorii nu sunt afectați de această funcționalitate și pot oricând să îți trimită mesaje private. Dacă crezi că un moderator te agresează în ciuda acestui lucru, trebuie să contactezi [echipa noastră de asistență](mailto:support@ppy.sh).**
+
+1. Fă clic pe butonul de `Options` în meniul principal sau apasă `Ctrl` + `O`.
+2. Scrie `block` pentru a filtra opțiunea în secțiunea de `Mesaje în cadrul jocului`.
+3. Fă clic pe setarea care apare pentru a o activa.
+
+Utilizatorii care nu sunt pe lista ta de prieteni nu vor mai putea să te contacteze.
+
+#### Împiedică mesajele private noi de pe forum să ajungă la tine. {id=ignore-strangers-website}
+
+Du-te în [setările contului](https://osu.ppy.sh/home/account/edit) și fă clic pe `blochează mesajele private de la oameni care nu sunt pe lista ta de prieteni` în secțiunea de Confidențialitate. Mesajele private de la utilizatorii de pe lista ta de prieteni vor putea să îți trimită mesaje.
+
+### Și dacă încă întâmpin probleme cu un utilizator după ce l-am blocat? {id=ignore-did-not-help}
+
+**Fără griji, echipa noastră de asistență te va ajuta.**
+
+Dacă un utilizator încă te agresează, fie prin alte mijloace sau își folosește prietenii sau pe altcineva pentru a trece de eforturile tale de a îl bloca, te rugăm să ne trimiți un e-mail la [support@ppy.sh](mailto:support@ppy.sh).
+
+Asigură-te că incluzi orice jurnal cu mesaje sau alte dovezi în e-mail, iar echipa noastră de asistență va investiga cazul tău și îl va remedia pentru tine.
+
+### Ce sunt aceste pagini „Eu!” pe care le văd pe profilul altor utilizatori? {id=userpage}
+
+**Paginile Eu! sau pagini de utilizator sunt elemente speciale de profil la care au acces orice utilizator cu [suporter](https://osu.ppy.sh/home/support). Ele pot conține orice vrei — poze cu realizările tale, beatmap-urile tale, practic tot ce îți dorești!**
+
+Jucătorii cu [suporter](https://osu.ppy.sh/home/support) au privilegiul de a-și modifica propria lor pagină pentru a fi afișată pe profilul lor, pentru a fi văzut de toată lumea. Paginile Eu! pot conține orice vrei, atâta timp cât respectă [regulile comunității](/wiki/Rules)!
+
+Chiar dacă se întâmplă ca [suporterul](https://osu.ppy.sh/home/support) tău să se termine, pagina ta Eu! va **rămâne vizibilă** și vei putea în continuare să îi modifici conținutul.
```

Comment:
```suggestion
Chiar dacă se întâmplă ca statusul tău de [suporter](https://osu.ppy.sh/home/support) să se expire, pagina ta „eu!” va **rămâne vizibilă** și vei putea în continuare să îi modifici conținutul.
```

Reply:
isn't it supposed to be „să expire” and not „să se expire”?